solana/system_program.py

```diff
@@ -1,24 +1,24 @@
 """Library to interface with system programs."""
 from __future__ import annotations
 
-from typing import Any, List, NamedTuple, Union
+from typing import Any, List, NamedTuple, Union, cast
 
 from solana.instruction import InstructionLayout, decode_data, encode_data
 from solana.publickey import PublicKey
 from solana.transaction import AccountMeta, Transaction, TransactionInstruction
 
 # Instruction Indices
-_CREATE = 0
-_ASSIGN = 1
-_TRANSFER = 2
-_CREATE_WITH_SEED = 3
-_ADVANCE_NONCE_ACCOUNT = 4
-_WITHDRAW_NONCE_ACCOUNT = 5
-_INITIALZE_NONCE_ACCOUNT = 6
-_AUTHORIZE_NONCE_ACCOUNT = 7
-_ALLOCATE = 8
-_ALLOCATE_WITH_SEED = 9
-_ASSIGN_WITH_SEED = 10
+CREATE_IDX = 0
```

Comment:
I made this public to allow the tests to use it.

Reply:
I think we should still have these hidden from the user. If we need an index for tests, we can just disable whichever linter that complains in the test file, i.e.

```python
import solana.system_program as sp
sp._Transfer # pylint: disable=protected-access
```

Reply:
make it protected-access again.